pkg/client/client.go

```diff
@@ -76,19 +77,24 @@ func (c *Client) WriteMessage(ctx context.Context, channel string, msg interface
 // HandleChannel handles messages received in a given channel.
 func (c *Client) HandleChannel(channel string, handler func(ctx context.Context, body io.Reader) error) {
 	c.mux.HandleFunc("/channel/"+channel, func(w http.ResponseWriter, r *http.Request) {
-		l := logger.With(zap.String("operation", "write"), zap.String("channel", channel))
+		logger.Info("received request in client handle channel", zap.String("Channel", channel))
```

Comment:
zap string name should not be capitalized

Reply:
Also, I think it should be "on client handle.." instead of "in channel ..."

Reply:
okok